ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -19,8 +14,8 @@ import Box from '../../ui/box';
 import Button from '../../ui/button';
```

Comment:
You can remove this in favour of the new `Button` component from the `component-library` folder

Reply:
this is good to know, thanks! As an occasional visitors to your codebase, the different button components as part of multiple different design systems is a bit confusing. Maybe would help having a deprecation notice like in `ui/components/css/itcss` although I guess the whole `ui/` folder is not deprecated yet as it's widely used.

Reply:
never mind, just saw this fresh readme change in #18501 which addresses this, thanks!